src/transformers/searchservice/createDirectAnswer.ts

```diff
@@ -25,8 +28,16 @@ export function createDirectAnswer(data: any): FeaturedSnippetDirectAnswer | Fie
     };
   } else if (isFeaturedSnippetDirectAnswer) {
     const fieldType = commonDirectAnswerData.fieldType;
-    if (fieldType != BuiltInFieldType.MultiLineText && fieldType != BuiltInFieldType.RichText) {
-      throw new Error(`Unexpected fieldType for featured snippet direct answer: ${fieldType}`);
+    const supportedFieldTypes: BuiltInFieldType[] = [
```

Comment:
this is a super minor nit, but this constant won't change on each invocation of the function so it should probably be defined out of the scope of the function

Reply:
I just moved it outside the createDirectAnswer() method in the latest commit